community/thrift/APKBUILD

```diff
@@ -0,0 +1,222 @@
+# Contributor: Christopher Friedt <chrisfriedt@gmail.com>
+# Maintainer: Christopher Friedt <chrisfriedt@gmail.com>
+pkgname=thrift
+pkgver=0.11.0
+pkgrel=0
+pkgdesc="Thrift is a lightweight, language-independent software stack with an associated code generation mechanism for RPC"
+url="https://github.com/apache/${pkgname}/archive/${THRIFT_VER}.tar.gz"
```

Comment:
`url` is intended more as a home page than a link to the source.

Reply:
good point, thanks!